boards/riscv32/qemu_riscv32/qemu_riscv32.dts

```diff
@@ -1,23 +1,52 @@
 /dts-v1/;
 
-#include <riscv32-qemu.dtsi>
+#include <riscv32-fe310.dtsi>
 
 / {
-	model = "qemu_riscv32";
-	compatible = "qemu,riscv32";
-
+	model = "SiFive HiFive 1";
+	compatible = "sifive,hifive1";
 	aliases {
-		uart-0 = &uart0;
+		led0 = &led0;
```

Comment:
are there really led's in the qemu model?  Seems like we shouldn't have them in the dts file.